data/ui/StationView/ShipMarket.lua

```diff
@@ -106,6 +113,9 @@ local function buyShip (sos)
 	if def.hyperdriveClass > 0 then
 		player:AddEquip(Equipment.hyperspace["hyperdrive_" .. def.hyperdriveClass])
 	end
+	for _, e in pairs(manifest) do
+		player:AddEquip(e)
```

Comment:
Never mind, did a silly comment since my font is too small to read, please disregard.